roles/kubernetes/preinstall/tasks/0020-verify-settings.yml

```diff
@@ -347,3 +347,17 @@
   when:
     - containerd_config is defined
     - not ignore_assert_errors
+
+- name: Get current state of time synchronization
+  shell: "echo $(($(date +%s%N)/1000000))"
```

Comment:
The above line is invalid on ansible-lint as https://gitlab.com/kargo-ci/kubernetes-sigs-kubespray/-/jobs/1479709010
```
roles/kubernetes/preinstall/tasks/0020-verify-settings.yml:351: no-changed-when Commands should not change things if nothing needs doing
```
Please check https://ansible-lint.readthedocs.io/en/latest/default_rules.html#no-changed-when for fixing.

Reply:
Let me add "changed_when: false"